### PR TITLE
tipb: add sig for tidb builtin function TidbShard

### PR DESCRIPTION
**Pull request by TonsnakeLin**
Signed-off-by: TonsnakeLin <lpbgytong@163.com>

### What problem does this PR solve?
We are going to add a new bulitin function TidbShard(), so add the ScalarFuncSig to tipb first.

### What is changed and how it works?

What's Changed:
go-tipb/expression.pb.go 

How it Works:

### CODE REVIEW

**Comment by TonsnakeLin** [comment]
@cfzjywxk @youjiali1995 @MyonKeminta
Please review it quickly if have time, because I have the other PR depends on it.
Thanks !!!

File: proto/expression.proto [hunk 1]
```diff
@@ -425,6 +425,7 @@ enum ScalarFuncSig {
 	UUID = 4569;
 	VitessHash = 4570;
 	IsUUID = 4571;
+	TidbShard = 4572;
```
Comment (youjiali1995) [inline proto/expression.proto:428]:
```suggestion
	TiDBShard = 4572;
```

Comment (TonsnakeLin) [reply proto/expression.proto]: already fixed

**Review by youjiali1995 — APPROVED** [review]

**Review by cfzjywxk — APPROVED** [review]

**Pull request merged**